contrib/pom.xml

```diff
@@ -601,9 +601,9 @@
       <optional>true</optional>
     </dependency>
     <dependency>
-      <groupId>com.datatorrent</groupId>
-      <artifactId>dt-common</artifactId>
-      <version>${dt.framework.version}</version>
```

Comment:
I think it's better to use apex.core.version instead of just apex.version since apex technically includes malhar.